INSTALL.md

```diff
@@ -1,11 +1,15 @@
 Installing From Sources
 =======================
 
-This document presents instructions to install this branch of Coq.
-For more general installation instructions and information about known
-build system issues, please consult the wiki page: 
+If you are just a user, you should probably install Coq using the
+platform or a package manager (e.g. opam or Nix) instead.
```

Comment:
To oppose usages rather than classifying people, I would suggest:
```suggestion
The recommended way to install and use Coq is to install the Coq
platform or to rely on a package manager (e.g. opam or Nix).
```